### DISCUSSION TITLE
🦊 Version 9.1 for Firefox released

### DISCUSSION THREAD

**Discussion by paulrouget** [aclap-dev/video-downloadhelper#1095] [Announcements]
Version v9.1 for Firefox is available https://addons.mozilla.org/firefox/addon/video-downloadhelper

What's new:

### New feature: Download History 

<img width="386" alt="Screenshot 2024-07-26 at 14 25 54" src="https://github.com/user-attachments/assets/68e3cc23-47c6-4d47-b6f7-dc6f40ac9598">

Give it a try, and tell us how you like it :)

### Fixed

- many many locales updated! Thanks to all the contributors :)
- added a History feature
- fixed queue logic for `Download As` actions
- fixed non-functional smartnaming
- improved video detection for some websites
- fixed slow popup for some websites
- fixed download title for some websites

**Comment by juleppar** [comment]
Is it possible that %title property doesn't work with YouTube? It only seems to produce YouTube-www youtube

**Comment by dddux** [comment]
I would just like to tell you how awesome is this new Video-Downloadhelper. It is absolutely awesome. Using it in Debian Linux with Firefox ESR, You've really made the best downloading app there is. Functionality and looks, just awesome. All 4 thumbs up! :)

**Comment by Wilber2** [comment]
I'm using VDH 8.0.0.6 on FF 129.0 and I do not see a way to update to 9.1 the link has no update button that I see.

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.

Try Ctrl+Shift+a.  That opens the Firefox Addons Manager.  Then look for the VDH entry and . . .

![#01](https://github.com/user-attachments/assets/681a411b-4240-4c70-bec1-5c76587260d0)
![#02](https://github.com/user-attachments/assets/8f00c8d2-7acf-4f03-9630-4b174eb61d3e)

If that doesn't work, look in the Wiki.  There's a link to the Wiki at the top of every page of this forum.  It is the closest thing we have to a user manual.  You should get in the habit of referring to it regularly.  There are links in there to both the general availability version of VDH as well as the beta version.  Personally, I recommend the beta version but everybody makes their own decision on that.

Come back & tell us how you get on.
